.github/dependabot.yml

```
@@ -0,0 +1,20 @@
+    target-branch: "update_to_rails_6_1"
+    # Labels on pull requests for version updates only
+
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      # Check for npm updates on Sundays
+      day: "sunday"
+    # Labels on pull requests for security and version updates
+    labels:
+      - "npm dependencies"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)